src/stores/SecurityData.js

```diff
@@ -0,0 +1,51 @@
+import {observable, asMap, action} from 'mobx';
+import {uniq} from 'lodash';
+
+export default class SecurityData {
```

Comment:
Since there is nothing else but Issues which are requested via "/security/issues/" URL, I suggest to call this store SecurityIssues and make it and its structure look like Regions/Region stores